`Development`: Fix minor grammar and spelling issues in README.md (#4449)
ls1intum/Artemis: README.md

5. **[File upload exercises](https://docs.artemis.ase.in.tum.de/user/exercises/file-upload/)** with manual assessment
6. **Exam mode**: Instructors can create online exams with exercise variants, integrated plagiarism checks, test runs and student reviews. You can find more information on [Exam mode student features](https://artemis.ase.in.tum.de/features/students) and on [Exam mode instructor features](https://artemis.ase.in.tum.de/features/instructors).
7. **Grading**: Instructors can configure grade keys for courses and exams to automatically calculate grades and display them to students. Grades can be easily exported as csv files to upload them into university systems (such as Campus online)

> 8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leaderboard and allows students to rate the assessments. Students can complain or ask for more feedback.   

Reviewer: ```suggestion
8. **Assessment**: Artemis uses double-blind grading and structured grading criteria to improve consistency and fairness. It integrates an assessment training process (based on example submissions and example assessments defined by the instructor), has a grading leader board, and allows students to rate the assessments. Students can complain or ask for more feedback.   
```
![image](https://user-images.githubusercontent.com/65814168/144746779-c897662f-a2b9-4069-91e4-bcace5913abd.png)

> 9. **Discussion**: Instructors can post announcements. Students can ask questions, post comments and react to other posts. Tutors can filter unanswered questions.

Reviewer: ```suggestion
9. **Discussion**: Instructors can post announcements. Students can ask questions, post comments, and react to other posts. Tutors can filter unanswered questions.
```

10. **Notifications**: Artemis supports customizable web and email notifications. Users can enable and disable different notification types.
11. **Team exercises**: Instructors can configure team exercises with real time collaboration and dedicated tutors per team.
12. **Lectures**: Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings and exercises into lectures, and define learning goals.
13. **Integrated markdown editor**: Instructors define problem statements using Markdown to easily format the text, and to integrate images and links.

> 14. **Plagiarism checks**: Artemis integrates plagiarism checks for programming exercises (based on [JPlag](https://github.com/jplag/JPlag)), text exercises and modeling exercises. It allows notifying students about identified plagiarism. Students can review and comment on the allegation. 

Reviewer: ```suggestion
14. **Plagiarism checks**: Artemis integrates plagiarism checks for programming exercises (based on [JPlag](https://github.com/jplag/JPlag)), text exercises, and modeling exercises. It allows notifying students about identified plagiarism. Students can review and comment on the allegation. 
```

Author: I wasn't sure about the oxford commas, I think there are a few more places that we would have to set them to keep it consistent.
